app/src/main/java/com/taraniuk/github/api/paging_library/data/repository/InstantRepositoryApi.kt

```diff
@@ -0,0 +1,10 @@
+package com.taraniuk.github.api.paging_library.data.repository
+
+import com.taraniuk.github.api.paging_library.data.retrofit.model.Model
+import io.reactivex.Single
+
+interface InstantRepositoryApi {
+
+    fun getAll(page: Int, size: Int): Single<Model>
```

Comment:
https://translate.google.com.ua/?um=1&ie=UTF-8&hl=ru&client=tw-ob#view=home&op=translate&sl=en&tl=ru&text=get%20all
It's perfect but need more clarification. What is "All"? Rhinos, hippos? The function name should be specific.

Reply:
Done